apps/picsa-tools/seasonal-calendar-tool/src/app/schema/schema_v0.ts

```diff
@@ -1,90 +1,76 @@
-import { IPicsaCollectionCreator } from '@picsa/shared/services/core/db_v2';
+import type { IPicsaCollectionCreator } from '@picsa/shared/services/core/db_v2';
 import { RxJsonSchema } from 'rxdb';
 
-  export interface MonthData {
-    month: string;
-    weather: string;
-  }
-  
-  interface CropMonth {
-    month: string;
-    activities: string[];
-  }
-  
-  export interface Crop {
-    name: string;
-    months: CropMonth[];
-    extraInformation: string;
-  }
-  
-  export interface CalendarDataEntry_v0 {
-    name: string;
-    timeAndConditions: MonthData[];
-    crops: Crop[];
-  }
+export interface CalendarDataEntry_V0 {
+  id: string;
+  name: string;
+  activities: {
+    [enterprise: string]: string[];
+  };
+  weather: string[];
+  meta: {
+    /** Specified time period. Sets number of items shown in activities and weather */
+    months: string[];
+    /** Selected enterprise type. Sets available enterprises (currently only 'crop') */
+    enterpriseType: 'crop';
+    /** Selected enterprises. Sets the entries shown in activities */
+    enterprises: string[];
+  };
+}
 
-  export const SCHEMA_V0: RxJsonSchema<CalendarDataEntry_v0> = {
-    title: 'calender entry schema',
-    version: 0,
-    keyCompression: false,
-    type: 'object',
-    properties: {
-      name: {
-        type: 'string',
-        default: '',
-      },
-      timeAndConditions: {
-        type: 'array',
-        items: {
-          type: 'object',
-          properties: {
-            name: {
-                type: 'string'
-            },
-            weather: {
-                type: 'string'
-            }
-          }
-        },
+// Example data to illustrate schema
+export const CALENDAR_ENTRY_MOCK_V0: CalendarDataEntry_V0 = {
+  id: 'mock_01',
+  name: 'Mock Entry',
+  activities: {
+    cassava: ['sowing', 'sowing', 'planting'],
+    maize: ['land_preparation', 'sowing', 'planting'],
+  },
+  weather: ['sunny', 'cloudy', 'rain'],
+  meta: {
+    months: ['march', 'april', 'may'],
+    enterpriseType: 'crop',
+    enterprises: ['cassava', 'maize'],
+  },
+};
+
+export const SCHEMA_V0: RxJsonSchema<CalendarDataEntry_V0> = {
```

Comment:
Amazing Schema @chrismclarke thanks for this refactor, it if more friendly to use